### PR TITLE
Fix the bug causing wait: Illegal option -n

### PR DESCRIPTION
**Pull request by feanors**
The proposed changes fixes the bug in issue https://github.com/arindas/mit-6.824-distributed-systems/issues/1

### CODE REVIEW

**Comment by arindas** [comment]
Umm you could add commits to this PR. No need to close this.

File: cmd/mr/scripts/test-mr.sh [hunk 1]
```diff
@@ -199,7 +199,14 @@ timeout -k 2s 180s ../../worker/mrworker ../../apps/early_exit/early_exit.so &
 # `jobs` ensures that any completed old processes from other tests
 # are not waited upon
 jobs &> /dev/null
-wait -n
+
+sum=`expr $(pgrep -c mrworker) + $(pgrep -c mrcoordinator)`
+newSum=`expr $(pgrep -c mrworker) + $(pgrep -c mrcoordinator)`
```
Comment (arindas) [inline cmd/mr/scripts/test-mr.sh:204]: Rename sum and newSum to initial_proc_count and proc_count

Directly initialise proc_count with initial_proc_count

File: cmd/mr/scripts/test-mr.sh [hunk 2]
```diff
@@ -199,7 +199,14 @@ timeout -k 2s 180s ../../worker/mrworker ../../apps/early_exit/early_exit.so &
...
+
+while [ $sum -eq $newSum ]
```
Comment (arindas) [inline cmd/mr/scripts/test-mr.sh:206]: Here add a comment, Busy wait loop: check if any background process has exited or not.
proc_count decreases by one when a process exits.

**Comment by arindas** [comment]
Its alright I'll add the comments later. Congrats on your first PR

**Comment by feanors** [comment]
Thanks I really appreciate it. I've deleted the repository that I've sent the PR from and was looking for ways to restore it, sorry for being a noob on this :D

Thanks again

**Pull request merged**